src/state/operations/reducer.ts

```diff
@@ -194,5 +206,104 @@ export default produce((draft: IOperationState, action: IAction<any, any>) => {
       delete draft.deleteOperationRow.error;
       break;
     }
+
+    case CREATE_OPERATION_LOADING: {
+      draft.create.status = "LOADING";
+      draft.create.hasSucceeded = false;
+      draft.create.isLoading = true;
```

Comment:
why do we use `hasSucceeded` and `isLoading`? they mimic the  `status`.
IMO, it would be easier just to rely on the status.

Reply:
I'm also ambarassed by this situation, isLoading and hasSucceeded should be normally computed fields. I agree with you to just rely on the status. But it will OK  to replace thoses fields with  computed fields.